.github/workflows/archive.yml

```diff
@@ -132,7 +132,7 @@ jobs:
         max_attempts: 3
         retry_on: error
         timeout_minutes: 30
-        command: cd main && yarn export --DRUPAL_CLIENT_ID ${{ secrets.PROD_DRUPAL_CLIENT_ID }} --DRUPAL_CLIENT_SECRET ${{ secrets.PROD_DRUPAL_CLIENT_SECRET }} --USE_REDIS=false
+        command: cd main && yarn export --DRUPAL_CLIENT_ID ${{ secrets.PROD_DRUPAL_CLIENT_ID }} --DRUPAL_CLIENT_SECRET ${{ secrets.PROD_DRUPAL_CLIENT_SECRET }}
```

Comment:
we'll still need to pass `false` to this command in some way.

package.json has: `"export": "yarn build:env-loader && node ./scripts/yarn/export.js --SSG --USE_REDIS",` so it will default to true when run in the workflows. this was fine before, because we still overrode it here

Locally, I was able to do that by adding: `.option('-nr, --no-USE_REDIS', 'Disable redis')` but you may know a cleaner way to negate boolean values

Reply:
This is exactly some of the interaction I was worried about with this. I wasn't sure if there were any situations where a flag was overriding another flag.

The solution you mention is the path forward. I didn't add the option because I added defaults (`false`) to all the boolean flags. But I assumed we'd eventually need to be able to explicitly negate them for some reason. I'm going to add to this PR the ability to negate all of them, and then fix this line in `archive.yml` as well.